tests/integration/mem_leak_test.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import libnmstate
+
+
+def test_repeat_call_of_show():
+    for _ in range(0, 1025):
```

Comment:
IMHO the 1025 should be explained or maybe even be calculated from `ulimit -n`.

Reply:
Changed to use `resource.getrlimit(resource.RLIMIT_NOFILE)`